### ISSUE TITLE
[BUG]: Esc key shows notice even when not processing any images e.g.: when closing setting tab

### ISSUE THREAD

**Issue by xRyul** [xRyul/obsidian-image-converter#79]

**Comment by juanbretti** [comment]
Yup, I have the same issue when pressing ESC key.
I get this warning:

```log
7[Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See <URL>
plugin:image-converter:34 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/feature/6662647093133312
eval @ plugin:image-converter:34
plugin:image-converter:34 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/feature/6662647093133312
eval @ plugin:image-converter:34
plugin:image-converter:34 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/feature/6662647093133312
eval @ plugin:image-converter:34
plugin:image-converter:34 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/feature/6662647093133312
eval @ plugin:image-converter:34
plugin:image-converter:34 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/feature/6662647093133312
eval @ plugin:image-converter:34
plugin:image-converter:34 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/feature/6662647093133312
eval @ plugin:image-converter:34
plugin:image-converter:34 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/feature/6662647093133312
```

**Comment by MaxWolf-01** [comment]
same. super annoying with vim since the latest update.

**Comment by xRyul** [comment]
Fixed in [1.2.1](https://github.com/xRyul/obsidian-image-converter/releases/tag/1.2.1). ESC now only works during conversion~

**Comment by xRyul** [comment]
> Yup, I have the same issue when pressing ESC key. I get this warning:
> 
> ```
> 7[Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See <URL>
> plugin:image-converter:34 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/feature/6662647093133312
> eval @ plugin:image-converter:34
> plugin:image-converter:34 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/feature/6662647093133312
> eval @ plugin:image-converter:34
> plugin:image-converter:34 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/feature/6662647093133312
> eval @ plugin:image-converter:34
> plugin:image-converter:34 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/feature/6662647093133312
> eval @ plugin:image-converter:34
> plugin:image-converter:34 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/feature/6662647093133312
> eval @ plugin:image-converter:34
> plugin:image-converter:34 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/feature/6662647093133312
> eval @ plugin:image-converter:34
> plugin:image-converter:34 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/feature/6662647093133312
> ```

This too! Fixed in [1.2.1](https://github.com/xRyul/obsidian-image-converter/releases/tag/1.2.1)

**Comment by juanbretti** [comment]
Fixed for me on [1.2.1](https://github.com/xRyul/obsidian-image-converter/releases/tag/1.2.1).
